### ISSUE TITLE
The "example queries" list is not accessable without Virginia Tech login credentials. (from vignette accessed by ?query_wos, clicking on hotlink for example queries brings me to an unaccessable page)

### ISSUE THREAD

**Issue by AustinDavidRichards** [vt-arc/wosr#15]

**Comment by AustinDavidRichards** [comment]
You're correct, I was trying to troubleshoot but realizing now it's not functional. Thanks.

**Comment by AustinDavidRichards** [comment]
Hi @crew102, I got off the phone with Clarivate, and according to them the API is the same as before, although there is a new requirement for registering with them to gain access, and they'll send a username and password, and this takes 10-14 days. Do you think that your package could be modified to work once access is gained?

**Comment by AustinDavidRichards** [comment]
Great!

I sent in a request for an account today and expect to hear back shortly, I’d be happy to keep you informed. Your package could be very useful to me. Thanks.

Austin Richards
US EPA
Program Analyst
Office of Research and Development
Office of Program Accountability and Resource Management
Richards.Austin@epa.gov<mailto:Richards.Austin@epa.gov>
Office: (202)564-3263
Cell: (202)893-6221

**Comment by crew102** [comment]
Yes, please do keep me informed. If you're able to get an account, I'll resubmit my request and try to get one for myself.

**Comment by AustinDavidRichards** [comment]
Hi Chris,

After months of waiting, I finally have an access code to the WOS API and am happy to report your package is in fact working! Although they’re upgrading to REST infrastructure, the SOAP keys are still working for now as expected and accessing all of the data.

Now, I’m wondering if you can provide any information on how difficult it would be to add a function which would return “citing articles”, ie, the citingArticles operation described here:

http://help.incites.clarivate.com/wosWebServicesExpanded/WebServiceOperationsGroup/WSPremiumOperations.html


Thanks!


Austin D. Richards
U.S. Environmental Protection Agency
Office of Research and Development
Office of Resource Management
Richards.Austin@epa.gov<mailto:Richards.Austin@epa.gov>
Office: (202)564-3263 | Cell: (202)893-6221

**Comment by crew102** [comment]
Hi Austin, so pull_wos() is working as expected now? I just thought that Clarivate had changed the design of their API, making wosr pretty much useless.

Unfortunately I can’t add any functionality beyond what currently exists, as I no longer have access to the API. I’ve tried to get temporary access from Clarivate for the purpose of development, but they haven’t responded to my requests.

**Comment by AustinDavidRichards** [comment]
Hi Chris,

Yes! WOSR is working as expected. Despite the eventual update to the API, the old infrastructure remains intact for now so the SOAP queries are working well. I wish I could help with the key – it took us months and months and countless phone calls to finally gain one.

Thanks,

Austin D. Richards
U.S. Environmental Protection Agency
Office of Research and Development
Office of Resource Management
Richards.Austin@epa.gov<mailto:Richards.Austin@epa.gov>
Office: (202)564-3263 | Cell: (202)893-6221

**Comment by crew102** [comment]
Oh wow, that’s great re: wosr being functional. I’ll try to add additional
functionality to that package if I ever end up getting a key.
